waku/v2/protocol/store/waku_store.go

```diff
@@ -328,8 +330,6 @@ func (store *WakuStore) fetchDBRecords(ctx context.Context) {
 	}
 
 	metrics.RecordMessage(ctx, "stored", store.messageQueue.Length())
-
-	store.log.Info(fmt.Sprintf("%d messages available in waku store", store.messageQueue.Length()))
```

Comment:
I merged this with the entry from the defered function above, just added the message count there as a field. Subtle difference is that now you get `messages=0` if the function fails out in the middle, but I thought that's fine.